### ISSUE TITLE
missing jquery.sparkline.min.js

### ISSUE THREAD

**Issue by MikhaelGerbet** [gwatts/jquery.sparkline#150]
Hi,
congratulation, this extention is greate !

But, for work with bower, we need file "jquery.sparkline.min.js" in your git.

Thank's for add ;)

**Comment by danyim** [comment]
I agree with this!

**Comment by hanssens** [comment]
Indeed, it would be great to not having to compile the plugin first in order to use it. By including the latest release files in the ~/dist folder, the plugin can be used with package managers like NPM and Bower without a sweat.

**Comment by hanssens** [comment]
... and @panzj seems to have created a [PR for this already](https://github.com/gwatts/jquery.sparkline/pull/153). Excellent!
